src/Magnum/Math/Test/CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ corrade_add_test(MathVectorTest VectorTest.cpp LIBRARIES MagnumMathTestLib)
 corrade_add_test(MathVector2Test Vector2Test.cpp LIBRARIES MagnumMathTestLib)
 corrade_add_test(MathVector3Test Vector3Test.cpp LIBRARIES MagnumMathTestLib)
 corrade_add_test(MathVector4Test Vector4Test.cpp LIBRARIES MagnumMathTestLib)
+corrade_add_test(MathVectorConstexprOps14 VectorConstexprOps14.cpp LIBRARIES MagnumMathTestLib)
+set_target_properties(
+    MathVectorConstexprOps14
+    PROPERTIES CORRADE_CXX_STANDARD 14)
```

Comment:
For various reasons I'm keeping compatibility with GCC 4.8, which doesn't support C++11 yet, so the C++14-only tests need to be compiled only if the compiler supports that. [Here is how it's done inside Corrade](https://github.com/mosra/corrade/blob/9e35d4eb31b60c8d0aaa2d0028eae469b59ed7b3/src/Corrade/Test/CMakeLists.txt#L61-L72), tho the condition should probably be put into some global place (a variable in `UseCorrade.cmake`?) and reused, instead of copypasting it thousand times.

Continuing with my suggestion of compiling the same test again, this would look like

```cmake
if(...) 
    corrade_add_test(MathVector2Cpp14Test Vector2Test.cpp LIBRARIES MagnumMathTestLib)
    ...
    set_target_properties(
        MathVector2Cpp14Test
        ...
        PROPERTIES CORRADE_CXX_STANDARD 14)
endif()
```